policyengine_us/variables/gov/irs/income/taxable_income/adjusted_gross_income/irs_gross_income/social_security/tax_unit_magi_excess.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_magi_excess(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    label = "Taxable Social Security modified adjusted gross income excess over base amount"
+    documentation = "Social security (OASDI) benefits included in AGI, including tier 1 railroad retirement benefits."
+    unit = USD
+    reference = "https://www.law.cornell.edu/uscode/text/26/86"
+
+    def formula(tax_unit, period, parameters):
+        ss = parameters(period).gov.irs.social_security.taxability
+        gross_ss = tax_unit("tax_unit_social_security", period)
+
+        # The legislation directs the usage an income definition that is
+        # a particularly modified AGI, plus half of gross Social Security
+        # payments. We assume that the 'half' here is the same underlying
+        # parameter as the lower taxability marginal rate (also 50% in the
+        # baseline), and that they would be mechanically the same parameter.
+
+        ss_fraction = ss.rate.lower * gross_ss
+        modified_agi_plus_half_ss = (
```

Comment:
Create a new PolicyEngine variable `tax_unit_combined_income_for_social_security_taxability` for this (in addition to this `tax_unit_magi_excess`, since it's used in this and `tax_unit_taxable_social_security`.

See https://www.ssa.gov/benefits/retirement/planner/taxes.html for the term _combined income_ being used, and [this ChatGPT conversation](https://chat.openai.com/share/a7575e4c-f6e4-4bcb-b3ec-1c5f8bc3f76f) for context on the naming.

```python
        return (
```